mobile/lib/features/wallet/receive_screen.dart

```diff
@@ -23,6 +24,16 @@ class ReceiveScreen extends StatelessWidget {
           const SizedBox(height: 50),
           SelectableText("Address: ${api.getNewAddress()}"),
           Text("Balance: ${balance.offChain} / ${balance.onChain}"),
+          ElevatedButton(
```

Comment:
Note: The commit that adds this as `Open Channel` button is outdated and I think it should be removed. 

The sentiment of `Create Invoice` is correct; that's in line with the flow where the channel gets automatically established upon initial payment.